src/app/settings/shelf-settings/components/edit-shelf-action-panel-configuration/edit-shelf-action-panel-configuration.component.ts

```diff
@@ -51,4 +55,46 @@ export class EditShelfActionPanelConfigurationComponent implements OnInit {
   public cancel(): void {
     this.onCancel.emit()
   }
+
+  public get formValidationTooltip(): string | undefined {
```

Comment:
This gets the tooltip for the save button when there is errors. That is not the same as `formValidationTooltip` so the name should be updated to reflect this

Reply:
I rename it to `saveButtonErrorsTooltip`